tests/src/system/basic/WskBasicTests.scala

```diff
@@ -87,14 +87,27 @@ class WskBasicTests
         stdout should include regex ("""(?i)whisk API version\s+v1""")
     }
 
+    it should "validate default property values" in {
+        var stdout = wsk.cli(Seq("property", "get", "--auth")).stdout
+        stdout should include regex ("""(?i)whisk auth\s*$""")      // default = empty string
+        stdout = wsk.cli(Seq("property", "get", "--apihost")).stdout
+        stdout should include regex ("""(?i)whisk API host\s*$""")  // default = empty string
+        stdout = wsk.cli(Seq("property", "get", "--namespace")).stdout
+        stdout should include regex ("""(?i)whisk namespace\s*_$""")// default = _
+    }
+
     it should "show api build version" in {
+        wsk.cli(wskprops.overrides ++ Seq("property", "set"))
         val stdout = wsk.cli(wskprops.overrides ++ Seq("property", "get", "--apibuild")).stdout
         stdout should include regex ("""(?i)whisk API build\s+201.*""")
+        wsk.cli(Seq("property", "unset", "--auth", "--apihost", "--apiversion", "--namespace"))
```

Comment:
All of the Scala tests override the setting for the property file, API host, namespace and auth. So you are right they don't have nor should they have any dependencies on a local file. The Java tests aren't so sound in that there are setting they may not be overriding completely.